File: Python/String/Minimum Number of Operations to Move All Balls to Each Box.py
class Solution:
    def minOperations(self, boxes: str) -> List[int]:
        ans=[]
        for i in range(len(boxes)):
            operation =0
            for j in range(len(boxes)):
                if boxes[j]=='1':
                    operation +=abs(i-j)
            ans.append(operation)
        return ans
        

File: CPP/String/Minimum Number of Operations to Move All Balls to Each Box.cpp
class Solution {
public:
    vector<int> minOperations(string boxes) {
      vector<int>ans;
      for(int i=0;i<boxes.size();i++)  {
        int operation =0;
        for(int j=0;j<boxes.size();j++){
            if(boxes[j]=='1'){
                operation+=abs(i-j);
            }
        }
        ans.push_back(operation);
      }
      return ans;
    }
};
